setup.py

```diff
@@ -36,7 +36,7 @@ def read(fname):
 
     # We don't declare our dependency on mxnet here because we build with
     # different packages for different variants (e.g. mxnet-mkl and mxnet-cu90).
-    install_requires=['sagemaker-container-support >= 1.0.0, <2'],
+    install_requires=['sagemaker-containers==2.1'],
```

Comment:
I think we should bump the package version for this package too. Github doesn't let me comment on line 15 -__-